notebooks/hybrid_rag_pipeline_with_breakpoints.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.        return document_store
</pre><p>An alternative could be to define the DocumentStore outside of this function and return the indexing pipeline. Similar to how hybrid_retrieval(doc_store) returns the query pipeline.</p><p> </p><p>Another advantage of that would be that users could set breakpoints in the indexing pipeline more easily too. If they want to try things out themselves.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I think it's good to separate it - this is just here because the pipeline needs a doc store